src/postformatting/rule-selector-remover.js

```diff
@@ -24,7 +23,7 @@ export default function ruleSelectorRemover (ast, selectorNodeMap, selectors) {
         }
       )
 
-      // NOTE: isCssSelectorRuleCritical mutates the rule
+      // remote the rule if no selector is left
```

Comment:
typo: remove

Reply:
It was fixed in this commit https://github.com/pocketjoso/penthouse/pull/222/commits/aaaf459e329076fe495cd90e23dcf23a4718ff9c